contrib/apelscripts/condor_batch.sh

```diff
@@ -23,5 +30,4 @@ TZ=GMT condor_history -constraint "$CONSTR" \
     -format "%d|" ResidentSetSize_RAW \
     -format "%d|" ImageSize_RAW \
     -format "%d|" RequestCpus \
-    "${HISTORY_SUFFIX}" > "$(condor_ce_config_val APEL_OUTPUT_DIR)/$output"
-
+    "${HISTORY_SUFFIX}" > $OUTPUT_FILE
```

Comment:
Same comment as before - it seems to me that `"${HISTORY_SUFFIX}"` should not be quoted as a single string.

If you think you need quoting for some reason (to escape whitespace or glob characters?), then consider constructing `HISTORY_SUFFIX` as an array and then quoting it as `"${HISTORY_SUFFIX[@]}"`

Reply:
(@matyasselmeci also suggests a less confusing variable name might also be `HISTORY_EXTRA_ARGS`)